[quant] ReflectionPad2d

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #48051 [quant][refactor] Reflection pad now uses AT_DISPATCH_FLOATING_AND_QINT_TYPES
* #48050 [quant] AT_DISPATCH_FLOATING_AND_QINT_TYPES
* #48037 [quant] out-variant for the reflection pad
* **#48036 [quant] ReflectionPad2d**

Differential Revision: [D25000347](https://our.internmc.facebook.com/intern/diff/D25000347)

Commit:
[quant] ReflectionPad2d

[ghstack-poisoned]

Commit:
Update on "[quant] ReflectionPad2d"


Differential Revision: [D25000347](https://our.internmc.facebook.com/intern/diff/D25000347)

[ghstack-poisoned]

Commit:
Update on "[quant] ReflectionPad2d"


Differential Revision: [D25000347](https://our.internmc.facebook.com/intern/diff/D25000347)

[ghstack-poisoned]

Commit:
Update on "[quant] ReflectionPad2d"


Differential Revision: [D25000347](https://our.internmc.facebook.com/intern/diff/D25000347)

[ghstack-poisoned]

jerryzh168 commented on aten/src/ATen/native/ReflectionPad.cpp:
```diff
@@ -346,23 +346,43 @@ void reflection_pad2d_out_template(
-        pad_l, pad_t);
-    });
+    if (input.is_quantized()) {
+      AT_DISPATCH_QINT_TYPES(input.scalar_type(), "qreflection_pad2d", [&] {
```
should we just add a dispatch that works for both floating types and qint types?

z-a-f replied:
Done!

jerryzh168 reviewed (approved):
looks good

Commit:
Update on "[quant] ReflectionPad2d"


Differential Revision: [D25000347](https://our.internmc.facebook.com/intern/diff/D25000347)

[ghstack-poisoned]
